src/main/java/se/bjurr/gitchangelog/api/GitChangelogApi.java

```diff
@@ -380,6 +379,12 @@ public GitChangelogApi withSettings(final URL url) {
     return this;
   }
 
+  /** {@link Settings}. */
+  public GitChangelogApi withSettings(final Settings settings) {
```

Comment:
I dont like to expose Settings in the API.

Reply:
Is there a better way to provide an existing Settings object?